SwiftFlowTests/CombinedReducerTests.swift

```diff
@@ -74,8 +74,7 @@ class CombinedReducerSpecs: QuickSpec {
                 let increaseByTwoReducer = IncreaseByTwoReducer()
 
                 let combinedReducer = CombinedReducer([increaseByOneReducer, increaseByTwoReducer])
-                let newState = combinedReducer._handleAction(CounterState(),
-                    action: StandardAction(emptyAction)) as? CounterState
+                let newState = combinedReducer._handleAction(StandardAction(emptyAction), state: CounterState()) as? CounterState
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 129 characters (line_length)